yb-voyager/cmd/analyzeSchema.go

```diff
@@ -402,6 +417,148 @@ func checkStmtsUsingParser(sqlInfoArr []sqlInfo, fpath string, objType string) {
 	}
 }
 
+var unsupportedIndexDatatypes = []string{
```

Comment:
I see a lot more in https://docs.yugabyte.com/stable/api/ysql/datatypes/ listed as 1(Table columns of this type cannot be part of an INDEX KEY.) Let's also check those?

Reply:
Hmm, I also thought of finding such doc but I was looking in CREATE INDEX one. 
But I think we can report these other ones later in a separate one as it might take some to understand the handling of some of these might need some thinking like`BIT [n] / Bit varying [n]` ones. Maybe I will see what all I can add to this list and report other ones in follow-ups of these.

Reply:
Yeah, sure, pls create a ticket, we can take this up later.

Reply:
I added all of these mostly as I just had to add them to the list and they worked just checked bit and bit varying with parser it was also straight forward.
Please take a look @makalaaneesh

Reply:
@priyanshi-yb nice! did you actually verify that creating index on those types fails? In case docs have not been updated...

Reply:
I haven't tried all the types in the list, just the ones marked as 1 (and says Table columns of this type cannot be part of an INDEX KEY) some of these e.g. CIRCLE, POLYGON also fails on PG with `btree` access method but still I have kept them in the list and will work in next PR on understanding if there is any other of creating indexes on these types in PG and whether that also fails in YB.